<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VEMainWindow</class>
 <widget class="QWidget" name="VEMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simple Vector Graphics Editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>9</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>41</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QSpinBox" name="spinBoxPenWidth">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Border width&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QToolButton" name="toolBtnSelect">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shape handling: Select, move and transform&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/resources/arrow_icon.png</normaloff>:/icons/resources/arrow_icon.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="toolBtnPLine">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Polyline: Draw a Polyline&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>L</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/resources/poly_line_icon.png</normaloff>:/icons/resources/poly_line_icon.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="toolBtnRect">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rectangle: Draw a rectangle&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>R</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/resources/rectangle_icon.png</normaloff>:/icons/resources/rectangle_icon.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBtnBrushColor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBtnSave">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save to File&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/resources/save_icon.png</normaloff>:/icons/resources/save_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBtnLoad">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open File&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/resources/folder_icon.png</normaloff>:/icons/resources/folder_icon.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBtnPenColor">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>graphicsView</tabstop>
  <tabstop>pushBtnLoad</tabstop>
  <tabstop>pushBtnSave</tabstop>
  <tabstop>toolBtnSelect</tabstop>
  <tabstop>toolBtnPLine</tabstop>
  <tabstop>toolBtnRect</tabstop>
  <tabstop>spinBoxPenWidth</tabstop>
  <tabstop>pushBtnBrushColor</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
